src/CreateVFE.tsx

```diff
@@ -57,6 +59,19 @@ function CreateVFEForm({ onCreateVFE }: CreateVFEFormProps) {
     }
   }
 
+  function handleAudioChange(e: React.ChangeEvent<HTMLInputElement>) {
+    const file = e.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = () => {
+        const audioURL = reader.result as string; // Assuming the result is a string representing the URL
+        setAudio(audioURL); // Set audio state to the selected file URL
+      };
+      reader.readAsDataURL(file);
```

Comment:
The way we load images/videos/audio has changed to be more performant.
This can be simplified to the following now:
```js
if (file) {
  setAudio(URL.createObjectURL(file));
}
```